tutorials/dataframe/df006_ranges.C

```diff
@@ -43,19 +43,19 @@ int df006_ranges()
    auto c_all = d.Count();
 
    // This is how you can express a range of the first 30 entries
-   auto d_0_30 = d.Range(0, 30);
+   auto d_0_30 = d.Range(0, 30);  // you can also get the first 30 entries with `d.Range(30)`
```

Comment:
why don't we just use `Range(30)`? (and similarly `Range(50)` below)

Reply:
Good point. I was wondering the same but thought that it was maybe meant to emphasize that Range takes the values between an initial value and an end value and that the default starting value is 0. Anyways, I fixed it in a second commit.